securitas/templates/groups.html

```diff
@@ -1,14 +1,9 @@
-{% extends "with_navbar.html" %}
+{% extends "main.html" %}
```

Comment:
This probably still needs more styling.

Reply:
https://github.com/fedora-infra/securitas/pull/92/commits/1b706823cc76659202b2ed6ba18b63016697f178